src/components/ui/font-family-toolbar-button.tsx

```
@@ -0,0 +1,88 @@
+'use client';
+
+import * as React from 'react';
+import { useEditorPlugin, useEditorSelector } from '@platejs/core/react';
+import { FontFamilyPlugin } from '@platejs/basic-styles/react';
+import { ChevronDown } from 'lucide-react';
```

ESLint

[error] 'ChevronDown' is defined but never used.

(@typescript-eslint/no-unused-vars)